Tasks/AzureRmWebAppDeploymentV3/package.json

```diff
@@ -22,7 +22,7 @@
     "@types/q": "1.0.7",
     "archiver": "1.2.0",
     "azure-pipelines-task-lib": "^3.4.0",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.220.2",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "3.220.2",
```

Comment:
Publish azure-pipelines-tasks-azure-arm-rest-v2 and update to 3.221.0

Reply:
Done